boards/arm/sam_e70_xplained/sam_e70_xplained-common.dtsi

```diff
@@ -157,4 +157,8 @@ zephyr_udc0: &usbhs {
 	pinctrl-0 = <&pb3a_can0_rx0 &pb2a_can0_tx0>;
 	bus-speed = <125000>;
 	bus-speed-data = <1000000>;
+
+	can-transceiver {
+		max-bitrate = <5000000>;
+	};
```

Comment:
Hi @henrikbrixandersen ,
The sam_v71_xult board have a 5Mbit can-fd transceiver too.
Do you mind add can entry for sam_v71_xult board?

Reply:
@nandojve Added, thank you.

Reply:
Thank you for this PR!